specs/mediawiki/v1/content.yaml

```diff
@@ -202,6 +236,29 @@ paths:
         # work in current Chrome and Firefox.
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}
+      x-amples:
+        - title: Get html by title from Parsoid
```

Comment:
Nope. this one is going to the storage, there are no `no-cache` headers. Invert the two tests - fetching from Parsoid should come first